src/sentry/silo/README.md

```diff
@@ -12,43 +12,53 @@ For SaaS deployment, we want to introduce sensitive data residency as part of th
 
 To set up the region silos locally, you'll need to make a few changes:
 
-In `~/.sentry/sentry.conf.py`, add the following:
+1. In `~/.sentry/sentry.conf.py`, add the following:
```

Comment:
With the new devserver mode you shouldn't need to add this information to your configuration file. It may be necessary for ngrok aliases though 🤔

Reply:
Split up the ngrok instructions from the regular one, they'll need to set `SENTRY_SUBNET_SECRET` for webhooks, and `SENTRY_MONOLITH_REGION` so that the split db script updates their `--monolith--` mappings.